.gitignore

```diff
@@ -1,3 +1,5 @@
+package-lock.json
+
```

Comment:
```suggestion
```

Reply:
Let it remain there. It doesn't do harm and will be useful for when one needs to run `npm audit` (eg: in our use case), only `.npmrc` file would need to be updated temporarily (to allow generation of `package-lock.json`).

Reply:
Marking it resolved based on the reason mentioned.

Reply:
No.

Reply:
Come on, now you're just being difficult :see_no_evil:

Reply:
Updated with https://github.com/papandreou/express-processimage/commit/9d825c7498e78ea2026f7eb1bbde4645193bf238